recipes-core/tdx-enc-handler/tdx-enc-handler/tdx-enc-caam.sh

```diff
@@ -61,6 +61,20 @@ tdx_enc_key_gen() {
     fi
 }
 
+# backup original data in partition (if not encrypted)
+tdx_enc_backup_data() {
+    mkdir -p "${TDX_ENC_STORAGE_MOUNTPOINT}"
+    if mount -t ext4 ${TDX_ENC_STORAGE_LOCATION} "${TDX_ENC_STORAGE_MOUNTPOINT}"; then
+        tdx_enc_log "Backing up original content..."
+        
+        mkdir /tmp/${TDX_ENC_DM_DEVICE}
+        chmod 777 /tmp/${TDX_ENC_DM_DEVICE}
```

Comment:
Why `chmod 777` here?

Reply:
No reason, this was originall copied from elsewhere.
Have removed this in my local changes and will test it today.

Reply:
@sergioprado - Are you happy to resolve this one as the latest code has this change.